modules/ROOT/pages/csrf-security.adoc

```diff
@@ -1,7 +1,7 @@
 = CSRF security
 :description: Bonita REST API is secured against Cross-Site Request Forgery (CSRF) attacks.
 
-Bonita REST API is secured against Cross-Site Request Forgery (CSRF) attacks.
+{description}
 
 CSRF (also known as XSRF and Sea-Surf), is an example of a malicious attack on a trusted website.
 While the user is authenticated, by being logged on to a trusted website, malicious requests are transmitted without the user knowing, or giving their consent.
```

Comment:
by being logged on to a trusted website => by being logged on a trusted website

I may be wrong, but it was difficult to read the phrase at once.

Reply:
I simplified the phrase to "While the user is authenticated through a trusted website, malicious requests are transmitted without the user knowing or giving his consent."